cloudbuild.yaml

```diff
@@ -0,0 +1,75 @@
+steps:
+  - id: 'fetch_secret'
+    name: gcr.io/cloud-builders/gcloud
+    entrypoint: 'bash'
+    args: [ '-c', "gcloud secrets versions access latest --secret=stargate-artifactory --format='get(payload.data)' | tr '_-' '/+' | base64 -d > /workspace/ci/custom-settings.xml" ]
```

Comment:
Will this have the same concern with forked repo builds that we have now? I mean will forks be able to discover secrets?

Reply:
Yes but that's an issue that many other repos face as well. This step comes directly from their docs

https://cloud.google.com/cloud-build/docs/securing-builds/use-encrypted-secrets-credentials#example_build_request_using_an_encrypted_variable

Reply:
This will now run automatically on every PR made by a contributor/owner and can be triggered on PRs from forks with `/gcbrun`